scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -381,7 +381,7 @@
         (let [has-gpus? (boolean (or (> (or (.getScalarValue (.getCurrAvailableResources target-vm) "gpus") 0.0) 0)
                                      (some (fn gpu-task? [req]
                                              @(job-needs-gpus? (:job req)))
-                                           (concat (.getRunningTasks target-vm)
+                                           (into (vec (.getRunningTasks target-vm))
```

Comment:
I can't put a comment on it, but I think something may have went wrong with a merge at some point. This gpu-host-constraint should be in cook.mesos.constraints.